test/e2e/framework.go

```diff
@@ -101,7 +101,7 @@ const (
 	busyboxImage        = "projects.registry.vmware.com/library/busybox"
 	nginxImage          = "projects.registry.vmware.com/antrea/nginx"
 	perftoolImage       = "projects.registry.vmware.com/antrea/perftool"
-	ipfixCollectorImage = "projects.registry.vmware.com/antrea/ipfix-collector:v0.4.7"
+	ipfixCollectorImage = "projects.registry.vmware.com/antrea/ipfix-collector:v0.5.1"
```

Comment:
since the main branch is using go-ipfix v0.5.2, we should change image to v0.5.2 too. We also need to change image [here](https://github.com/antrea-io/antrea/blob/fed9ae242c0e5f4e7ff6d3416259d91be895a485/ci/kind/test-e2e-kind.sh#L106)

Reply:
Updated